sunbeam-python/requirements.txt

```diff
@@ -45,3 +45,6 @@ python-libmaas
 
 # Faillible management
 tenacity
+
+# timeout
+timeout-decorator
```

Comment:
Initially I am sceptical to use this project as the last contribution is 5 years ago and there are no responses for the issues raised, however the package is available in ubuntu universe and so should be ok.

Reply:
I'm also uneasy about this. I looked at how it's being used, and I think both cases could be refactored to use tenacity retry to handle it gracefully, which would be nicer as tenacity is already a dependency

Reply:
Refactored to use tenacity.